test/PowerShellEditorServices.Test.Shared/Completion/CompletionExamples.psm1

```diff
@@ -10,10 +10,10 @@ Get-So
 $testVar
 
 Import-Module PowerShellGet
-Install-Mo
```

Comment:
This test here I assume to check that PSES understands imports for reference finding. The problem is that PowerShellGet I think is available by default in Windows PowerShell, so I don't think this ever actually worked. So I changed it to `Import-Module`.

Reply:
I've now changed this to `Get-Random`, since apparently that is invariant across all the PowerShell versions we support